verifyFiles.py

```diff
@@ -369,6 +446,21 @@ def main():
             metric_file = open(metrics[metric_name]["output_file"],'w')
 	    metric_file.write(pretty_string(metric_values[metric_name],conditions_dict))
 	    metric_file.close()
+	if args.execFile is not None:
+	  log_info("Writing executable details to file")
+          exec_file = open(args.execFile,'w')
+          for key in dictionary_executables:
+            executable_details_list=dictionary_executables[key]		
+	    if executable_details_list:
+	      str_file ="File:"
+	      #Below condition is in order to distinguish the intra-condition runs and append ** to the executable details. 
+	      if executable_details_list[0] == True:
```

Comment:
You could remove this statement based on the previous comment about inter vs intra condition differences.